rules/style.js

```diff
@@ -121,7 +121,11 @@ module.exports = {
     "operator-linebreak": [2, "after"],
     // Enforce padding within blocks
     // http://eslint.org/docs/rules/padded-blocks
-    "padded-blocks": [2, "never"],
+    "padded-blocks": [2, {
+      "blocks": "never",
+      "classes": "always",
```

Comment:
Scanning over our classes, it generally seems like we don't have use padding on classing. With some minor exceptions. Seems like both `always` and `never` would for us to do an update.

The human-friendly renaming might not be appropriate for this particular PR.